tools/msvs/vswhere_usability_wrapper.cmd

```diff
@@ -1,20 +1,32 @@
 :: Copyright 2017 - Refael Ackermann
 :: Distributed under MIT style license
 :: See accompanying file LICENSE at https://github.com/node4good/windows-autoconf
-:: version: 1.14.0
+:: version: 1.15.4
 
 @if not defined DEBUG_HELPER @ECHO OFF
 setlocal
+set "InstallerPath=%ProgramFiles(x86)%\Microsoft Visual Studio\Installer"
+if not exist "%InstallerPath%" set "InstallerPath=%ProgramFiles%\Microsoft Visual Studio\Installer"
+if not exist "%InstallerPath%" goto :no-vswhere
+:: Manipulate %Path% for easier " handeling
+set Path=%Path%;%InstallerPath%
```

Comment:
I usually don't like adding stuff "just to be safe", but since I don't know exactly why or how this might break (but it probably will for someone), I added them.